tower_cli/api.py

```diff
@@ -262,9 +266,27 @@ def test_mode(self):
                                      password='This is the best wine.',
                                      verbose=False, format='json'):
             adapters = copy.copy(self.adapters)
-            faux_adapter = FauxAdapter(
-                url_pattern=self.prefix.rstrip('/') + '%s',
-            )
+
+            class VersionedFauxAdapter(FauxAdapter):
```

Comment:
+1 here. I think this method does minimize the impact of mutable API version on our test suit.